rcljava/src/main/java/org/ros2/rcljava/client/Client.java

```diff
@@ -38,6 +38,10 @@
   
   <V extends MessageDefinition> boolean removePendingRequest(ResponseFuture<V> future);
 
+  long prunePendingRequests();
+
+  long prunePendingRequestsOlderThan(long nanoTime);
```

Comment:
Adding some docstrings describing these methods would be nice.

Reply:
Added in https://github.com/osrf/ros2_java/pull/82/commits/24fbf81821e3e33fbcc2fb0602ab5dc8bd1e6aab.